packaging/linux/common/khiops_env/khiops_env.in

```diff
@@ -57,7 +57,11 @@ get_script_dir() {
 
 # Setting up mpi for multiple machines
 if [[ -n $KHIOPS_MPI_HOST_FILE ]]; then
-    MPI_EXTRA_FLAGS="@MPIEXEC_HOSTFILE_FLAG@ $KHIOPS_MPI_HOST_FILE"
+    _MPI_EXTRA_FLAGS="@MPIEXEC_HOSTFILE_FLAG@ $KHIOPS_MPI_HOST_FILE"
+    # Test that @MPIEXEC_HOSTFILE_FLAG@ is not empty
```

Comment:
I'm not convinced by the mechanism:
- the comment will be weird on macos. It will be "# Test that is not empty" because MPIEXEC_HOSTFILE_FLAG will be replace by the empty string.
- we understand it only if we look at install.cmake

Reply:
Ok, right. As we currently Only Conda is supported on MacOS, the host file is not relevant in this context anyway. Hence, the `-host` / `-f` combination is not legit in this context.
Hence, I will simplify to only add the `host` on MacOS with MPICH.

Reply:
Done.